tutorials/README.md

```diff
@@ -0,0 +1,44 @@
+# Tutorials
+
+## iOS + Android
+* [Getting started](https://github.com/JetBrains/compose-multiplatform-ios-android-template)
+
+## Desktop
+* [Getting started](https://github.com/JetBrains/compose-multiplatform-desktop-template)
+* [Image and icon manipulations](Image_And_Icons_Manipulations)
+* [Mouse events and hover](Mouse_Events)
+* [Scrolling and scrollbars](Desktop_Components#scrollbars)
+* [Tooltips](Desktop_Components#tooltips)
+* [Context Menu](Context_Menu/README.md)
+* [Top level windows management](Window_API_new)
+* [Menu, tray, notifications](Tray_Notifications_MenuBar_new)
+* [Keyboard support](Keyboard)
+* [Tab focus navigation](Tab_Navigation)
+* [Swing interoperability](Swing_Integration)
+* [Navigation](Navigation)
+* [Accessibility](Accessibility)
+* [Building a native distribution](Native_distributions_and_local_execution)
+
+## Web
+* [Getting started](https://kotl.in/wasm-compose-example)
+
+## General
+
+See [Foundation](https://developer.android.com/jetpack/compose/documentation#core) and [Design](https://developer.android.com/jetpack/compose/documentation#design) tutorials from Google. They were originally written for Android, but most of information applies for the other targets as well.
+
+## Libraries
+
+### HTML
+Compose HTML Library allows you to build reactive user interfaces for the web in Kotlin, using the concepts and APIs of Compose to express the state, behaviour, and logic of your application.
```

Comment:
```suggestion
The Compose HTML library allows you to build reactive user interfaces for the web in Kotlin using the concepts and APIs of Compose to express the state, behavior, and logic of your application.
```

Reply:
Please replace in other places `Compose HTML Library` with `the Compose HTML library`. We should capitalize only the name of the library and use an article.

Reply:
Done